help/templates/template.cshtml

```diff
@@ -97,7 +100,7 @@
 															<a href="/core.html">Core</a>
 															<ul>
 																<li><a href="/core-targets.html">Targets</a></li>
-																<li><a href="/apidocs/v5/fake-core-environment.html.html">Environment</a></li>
+																<li><a href="/apidocs/v5/fake-core-environment.html">Environment</a></li>
```

Comment:
good eyes :)

Reply:
I had to click on every link to see if i missed something ;)